content/docs/guides/components/modeldb.md

```diff
@@ -0,0 +1,94 @@
++++
+title = "ModelDB"
+description ="ModelDB - A system to manage machine learning models"
+weight = 10
+toc = true
+[menu]
+[menu.docs]
+  parent = "components"
+  weight = 5
++++
+## Deploying ModelDB
+
+Use the below commands to deploy ModelDB.
+
+```
+ks generate modeldb modeldb
+ks apply default -c modeldb
+```
+
+## Using ModelDB
```

Comment:
Maybe rename this section to Data Model?

Reply:
Actually maybe just call it concepts.

You can then add other concepts like Dataset, Model, Metrics, etc... see comment below